esmvaltool/utils/draft_release_notes.py

```diff
@@ -41,7 +41,7 @@
 }
 
 PREVIOUS_RELEASE = {
-    'esmvalcore': datetime.datetime(2021, 7, 23, 00),
+    'esmvalcore': datetime.datetime(2021, 6, 14, 00),
```

Comment:
hwat? This was indeed done on July 23rd, see https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.3.1 - are you wanting to use the 2.3.0 date?